app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -111,6 +111,11 @@ ShortFormNavigationService = ($state) ->
           'household-members'
         else
           'household-intro'
+      when 'general-lottery-notice'
+        'status-programs'
+      when 'status-vouchers'
```

Comment:
@kaptron I like what you did -- it was not possible for me to inject Nav to Ctrl and Ctrl to Nav, but you did inject services and that made it much easier.

And yes we can make it a chore to refactor it further but this works! merging.